src/Compute/Compute.Test/ScenarioTests/VirtualMachineTests.ps1

```diff
@@ -7441,3 +7441,50 @@ function Test-VMTLWithGallerySourceImage
         Clean-ResourceGroup $rgname;
     }
 }
+
+
+<#
+.SYNOPSIS
+Windows machine test ensuring the EnableAutoUpdate value on the 
```

Comment:
Update the description so we understand what this test is for.

Reply:
done